src/Display/Modal/ModalStyle.ts

```diff
@@ -193,3 +131,87 @@ export const ModalFooter = styled.footer<{ isChildrenInMultiLines: boolean }>`
     `;
   }}
 `;
+
+export const ModalContentArea = styled.div<InnerProps>`
+  position: relative;
+  background: ${({ hideContentArea }) =>
+    hideContentArea ? 'transparent' : `${Greyscale.white}`};
+  margin: 120px auto;
+  max-width: 95vw;
+  outline: none;
+
+  ${({ size }) => {
+    switch (size) {
+      case 's':
+        return `
+          width: ${SIZES.s}px;
+        `;
+      case 'm':
+        return `
+          width: ${SIZES.m}px;
+        `;
+      case 'l':
+        return `
+          width: ${SIZES.l}px;
+        `;
+      case 'xl':
+        return `
+          width: ${SIZES.xl}px;
+        `;
+      default:
+        return `
+          width: ${SIZES.default}px;
+        `;
+    }
+  }}
+
+  ${({ centering }) => {
+    if (centering) {
+      return `
+        display: flex;
+        flex-direction: column;
+        max-height: 85vh;
+      `;
+    }
+  }}
+
+  ${({ fullscreen }) => {
+    if (fullscreen) {
+      return `
+        display: flex;
+        flex-direction: column;
+        max-width: 100vw;
+        max-height: 100vh;
+        width: 100vw;
+        height: 100vh;
+        margin: 0;
+
+        ${ModalBody} {
+          overflow: auto;
+        }
+
+        ${ModalFooter} {
+          margin-top: auto;
+        }
```

Comment:
This could also be accomplished by adding a `data-fullscreen` attribute to the `ModalContentArea`. Though I guess your way is more consistent with the existing implementation.

Reply:
Yep, that is the reason I didn't use attribute selector. :)